torch/testing/_internal/common_methods_invocations.py

```diff
@@ -750,6 +750,17 @@ def to_float(start, end, step):
     yield SampleInput(2)
     yield SampleInput(1, args=(3, 1))
 
+def sample_inputs_randn(op, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=False)
+
+    shapes = (
```

Comment:
should it also have 0-element input?

Reply:
Sure

Reply:
I'll add this in the follow-up PR adding `refs.rand_like`